src/ParallelAlgorithms/Amr/Projectors/Protocols/Mutator.hpp

```diff
@@ -37,6 +41,9 @@ namespace amr::protocols {
 ///   amr::Actions::InitializeChild and amr::Actions::InitializeParent hold the
```

Comment:
I missed this before, but you need a blank line before "The Mesh and Element..." two lines above here.